### PR TITLE
Posts

### PR DESCRIPTION
**Pull request by MaxHalleran**
Adds post support. Includes a Post index view and individual Post views. 

And by views, as this is an API, I mean API response. 

No database or auth yet.

Testing: 
1. `dotnet run`
2. Go to `localhost:5001/swagger`. This exposes the endpoints that are available to the application. 
3. Get posts, try it out, click 'execute'
4. Copy the 'id' of a post
5. Get /posts/{id}, try it out, paste id, click execute
6. Ensure response body has desired info

### CODE REVIEW

File: MyFirstBlog/.vscode/launch.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+            "name": ".NET Core Launch (web)",
+            "type": "coreclr",
+            "request": "launch",
+            "preLaunchTask": "build",
+            // If you have changed target frameworks, make sure to update the program path.
+            "program": "${workspaceFolder}/bin/Debug/net5.0/MyFirstBlog.dll",
+            "args": [],
+            "cwd": "${workspaceFolder}",
+            "stopAtEntry": false,
+            // Enable launching a web browser when ASP.NET Core starts. For more information: https://aka.ms/VSCode-CS-LaunchJson-WebBrowser
```
Comment (Nolan-E) [inline MyFirstBlog/.vscode/launch.json:17]: `.json` doesn't support comments. Might need to remove these.

**Review by Nolan-E — CHANGES_REQUESTED** [review]
Small change. Everything else seems fine to me

File: MyFirstBlog/.vscode/launch.json (hunk 1 above)
Comment (MaxHalleran) [reply MyFirstBlog/.vscode/launch.json:17]: I've removed the comments

**Review by Nolan-E — APPROVED** [review]
🌮

**Pull request merged**